### ISSUE TITLE
Search Mode Configured with newEditor or existingEditor is Missing Replace Field

### ISSUE THREAD

**Issue by RobertKlohr** [microsoft/vscode#116593]
- VS Code Version: 1.53.2
- OS Version: Windows 10

Steps to Reproduce:

1. Configure User settings search.mode to newEditor or existingEditor.
2. Trigger Search: Find in Files (Shift+Ctril+F) or Find in Folder (Shift-Alt+F).
3. The ">" next to the search field is missing and there is no way to open a Replace field. (See attached screenshot.)
4. Setting the search.mode back to view enables the Replace field.

Does this issue occur when all extensions are disabled?: Yes

![replace missing from search in editor](https://user-images.githubusercontent.com/39203582/107837832-af908b00-6d57-11eb-928c-c615e089f355.png)

**Comment by ArturoDent** [comment]
I assume that is intended behaviour as **search editors** never have had replace functionality (other than some special steps only within the results themselves).

**Comment by JacksonKearl** [comment]
duplicate of #93031

**Comment by RobertKlohr** [comment]
The reason I opened this is that the write-up for the new search feature seemed to imply (at least to me but I may have been reading too far between the lines) that there was now functional parity between the search in the view and search in the editor as they were now sharing a common search.mode setting and having separate key binding was no longer needed.

As this was marked as a duplicate of #93031 and that issue was closed last year due to not getting enough votes what would be the proper next step if I want to see if there was enough interest in adding this functionality?  Do I just create a new feature request?
